app/assets/javascripts/discourse/app/components/topic-timeline/scroll-area.js

```diff
@@ -13,21 +16,23 @@ export default class TopicTimelineScrollArea extends GlimmerComponent {
   @tracked showButton = false;
   @tracked scrollPosition;
   @tracked current;
-  @tracked percentage = this._percentFor(
-    this.args.topic,
-    this.args.enteredIndex + 1
-  );
+  @tracked percentage = this.scrollPercentage;
   @tracked total;
   @tracked date;
   @tracked lastReadPercentage = null;
   @tracked position;
-  @tracked lastReadTop = Math.round(
-    this.lastReadPercentage * scrollareaHeight()
-  );
+  @tracked lastReadTop = this.lastReadHeight;
```

Comment:
Is this implementation correct? Combining `@tracked` and a `getter` feels off

Reply:
If this is the only place where `lastReadHeight` is used, you could inline it. But if not - it's totally ok 😃

Reply:
However, note that doing `blah=…` will only be evaluated during the first initialization of the component. If you want it to update live then it will need to be a getter.